src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/summarize_api.py

```diff
@@ -0,0 +1,188 @@
+import os
```

Comment:
to unite api and opset in `summarize.py` because of the same args and etc

To define by -t or --type =OP/API